trompace/mutations/person.py

```diff
@@ -90,7 +90,7 @@ def mutation_create_artist(artist_name: str, publisher: str, contributor: str, c
     if coverage:
         args["coverage"] = coverage
     if date:
-        args["date"] = date
+        args["date"] = _Neo4jDate(date)
```

Comment:
Instead of creating this object here, what about we specify that the type of this argument should be a `datetime.date` or `datetime.datetime` object? This way, we can render the formatted date in the `mutations.make_parameters` method.

Reply:
The date type is connected to the attributes of the types in the dataset, not to the type we try to post as parameter. So I don't think your proposal is a good idea. The second argument is that a date or datetime type does not support only a year or year+month formats.